.dev/assets/shared/js/frontend/components/sticky-header.js

```diff
@@ -0,0 +1,46 @@
+const stickyHeader = () => {
```

Comment:
@EvanHerman I'm hesitant to add JS here? If we absolutely need to let's avoid jQuery entirely since it will be removed in https://github.com/godaddy-wordpress/go/pull/722